app/styles/ooyala/_article-featured-video.scss

```diff
@@ -1,28 +1,128 @@
 .article-featured-video {
+	position: relative;
+	padding-bottom: 56.25%;
+
 	.video-container {
-		background-size: cover;
-		padding-top: 56.25%;
-		position: relative;
+		position: absolute;
 
 		&.player-ready {
 			.video-play-circle {
 				opacity: 1;
 			}
 
 			.video-details {
-				display: block;
+				visibility: visible;
 			}
 		}
 
 		&.is-played {
 			.video-details {
-				display: none;
+				flex: 1 0 70%;
+				visibility: hidden;
 			}
 
 			.ooyala-article-video {
 				display: block;
 			}
 		}
+
+		&.fixed {
+			@keyframes video-slide {
+				0% {
+					transform: translateY(-100%);
+				}
+				100% {
+					transform: translateY(0%);
+				}
+			}
+
+			align-items: center;
+			animation: video-slide .5s forwards;
+			background: $wds-color-white;
+			box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.2);
+			display: flex;
+			padding: 12px;
+			position: fixed;
+			transform: translateY(-100%);
+			top: $wds-global-navigation-height;
+			width: 100%;
+			z-index: $z-site-head - 1;
+
+			.wds-spinner {
+				display: none;
```

Comment:
why do we have to hide spinner here? Maybe condition is wrong and it shouldn't be active in this state?